model_compression_toolkit/qat/keras/quantizer/quant_utils.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2021 Sony Semiconductors Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import tensorflow as tf
+from model_compression_toolkit.core.common.constants import MIN_THRESHOLD
+from typing import Tuple
+
+
+def ste_ceil(x: tf.Tensor) -> tf.Tensor:
```

Comment:
This looks very similar to GPTQ maybe it should move to core?

Reply:
it's identical, but core is definately not the place for it (IMO). Maybe a keras common for GPTQ and QAT? Or just let it be separate and diverge over time.